Build failure with .NET 9 RC2

```
dotnet build -nr:false  --configuration Debug
    C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Retry.Tests\IceRpc.Retry.Tests.csproj : error NU1107:
      Version conflict detected for Microsoft.Extensions.DependencyInjection.Abstractions. Install/reference Microsoft.Extensions.DependencyInjection.Abstractions 8.0.0 directly to project IceRpc.Retry.Tests to resolve this issue.
       IceRpc.Retry.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.Options 8.0.2 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.0)
       IceRpc.Retry.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.DependencyInjection 8.0.1 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.2).
    C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Conformance.Tests\IceRpc.Conformance.Tests.csproj : error NU1107:
      Version conflict detected for Microsoft.Extensions.DependencyInjection.Abstractions. Install/reference Microsoft.Extensions.DependencyInjection.Abstractions 8.0.0 directly to project IceRpc.Conformance.Tests to resolve this issue.
       IceRpc.Conformance.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.Options 8.0.2 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.0)
       IceRpc.Conformance.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.DependencyInjection 8.0.1 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.2).
    C:\Users\jose\source\repos\icerpc-csharp\tests\IceRpc.Logger.Tests\IceRpc.Logger.Tests.csproj : error NU1107:
      Version conflict detected for Microsoft.Extensions.DependencyInjection.Abstractions. Install/reference Microsoft.Extensions.DependencyInjection.Abstractions 8.0.0 directly to project IceRpc.Logger.Tests to resolve this issue.
       IceRpc.Logger.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.Options 8.0.2 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.0)
       IceRpc.Logger.Tests -> Microsoft.Extensions.Logging 8.0.1 -> Microsoft.Extensions.DependencyInjection 8.0.1 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 8.0.2).
```

--- a/tests/IceRpc.Conformance.Tests/IceRpc.Conformance.Tests.csproj
+++ b/tests/IceRpc.Conformance.Tests/IceRpc.Conformance.Tests.csproj
@@ -16,6 +16,8 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.*" />
+    <!-- Required to avoid version conflicts: see #4088 -->
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.*" />
     <PackageReference Include="NUnit.Analyzers" Version="4.3.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>

--- a/tests/IceRpc.Logger.Tests/IceRpc.Logger.Tests.csproj
+++ b/tests/IceRpc.Logger.Tests/IceRpc.Logger.Tests.csproj
@@ -11,6 +11,8 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.*" />
+    <!-- Required to avoid version conflicts: see #4088 -->
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.*" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
     <PackageReference Include="NUnit.Analyzers" Version="4.3.0">
       <PrivateAssets>all</PrivateAssets>

--- a/tests/IceRpc.Retry.Tests/IceRpc.Retry.Tests.csproj
+++ b/tests/IceRpc.Retry.Tests/IceRpc.Retry.Tests.csproj
@@ -11,6 +11,8 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.*" />
+    <!-- Required to avoid version conflicts: see #4088 -->
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.*" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
     <PackageReference Include="NUnit.Analyzers" Version="4.3.0">
       <PrivateAssets>all</PrivateAssets>

